### DISCUSSION TITLE
Multi-region testing

### DISCUSSION THREAD

**Discussion by nicholas-c** [artilleryio/artillery#3404] [Artillery]
Hey Artillery community 👋 

We have a multi-region application we're trying to load test, and naturally we can test single region scalability via Artillery easily, however we'd like to test global scaling via a global load test. 

An option would be to run the artillery run-lambda command multiple times from each region, but that's not exactly a science, has anyone come across this requirement in the past?

**Comment by hassy** [comment]
Hi @nicholas-c 👋 - running multiple commands (one per region) is the way to go right now. It's a common use case and we're planning to improve it:

1. Allow multiple regions to be set via `--region`
2. Aggregate and present per-region metrics in the text report
3. Show per-region metrics in Artillery Cloud

No ETA on that right now though

**Reply by nicholas-c** [reply]
Thanks for the quick reply @hassy! Much appreciated for the information, we'll have a play around with running concurrent commands in multiple regions and aggregating in datadog or similar for now.

Thanks again!
